### ISSUE TITLE
Formatted code in 1.0.0

### ISSUE THREAD

**Issue by bogomolov-dev** [vue-formation/vue-prismjs#3]
Is it possible to pass formatted code to the code property? I get a single line block.
In 0.1.1 it was not a problem by passing code to the slot.

```js
<prism language="json">{{row.item.response}}</prism>
```
vs.
```js
<prism language="json" :code="row.item.request"></prism>
```

**Comment by bhoriuchi** [comment]
Please provide a sample of formatted code

**Comment by bhoriuchi** [comment]
I think this may be a special case because you are passing an object and vue pretty prints objects when it interpolates them. You can just JSON.stringify(row.item.request, null, ‘. ‘) your code or I can add some sort of option to automatically do it

**Comment by bogomolov-dev** [comment]
Hi @bhoriuchi,
I handle the JSON object with JSON.stringify.

The class was filled by a AJAX call:
```js
export default class Profile {

  constructor (id, request, response) {    
    this._id = id;
    this._request = JSON.stringify(request, null, 2);
    this._response = JSON.stringify(response, null, 2);
  }
}
```

The array with n profiles are shown in Bootstrap-Vue table rows. In one cell the JSON should be printed.
```js
<prism language="json" :code="row.item.request"></prism>
```
I think HTML attributes cannot have some breaks/new lines.

**Comment by bhoriuchi** [comment]
i have published v1.1.0 to npm to fix this. See release notes, but TLDR using `textContent` instead of `innerText` to set the code text fixed it. I also brought slots back.

**Comment by bhoriuchi** [comment]
Fixed please use current version v1.2.0
